services/application.js

```diff
@@ -144,7 +150,10 @@ class Application {
     async saveApplication(params, context) {
         verifySession(context)
             .verifyInitialized()
-            .verifyPermission(USER_PERMISSION_CONSTANTS.SUBMISSION_REQUEST.CREATE);
+        const userScope = await this.#getUserScope(context?.userInfo, USER_PERMISSION_CONSTANTS.SUBMISSION_REQUEST.CREATE);
```

Comment:
This should be just checking if the user is creator of the application, not create permission.

Reply:
For saveApplication API, it will be two scenarios. The codes is updated.
1. store the submission request if it does not exist; I think it should be the **Create** permission.
2. if the submission request existed, it checks only the creator.